client/src/components/User/UserBeaconSettings.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <b-row class="ml-3 mb-1">
+    <i class="pref-icon pt-1 fa fa-lg fa-broadcast-tower"/>
+    <div class="pref-content pr-1">
+      <a id="beacon-settings" href="javascript:void(0)"
+      ><b v-b-modal.modal-beacon v-localize>Manage Beacon</b></a
+      >
+      <div v-localize class="form-text text-muted">Contribute variants to Beacon</div>
+      <b-modal
+          size="xl"
+          id="modal-beacon"
+          ref="modal"
+          ok-only
+          title="Manage Beacon"
+          title-tag="h1"
+          @show="onOpenModal">
+
+
+        <!-- Explanation text-->
+        <p>
+          The <a href="https://beacon-project.io">Global Alliance for Genomics & Health Beacon Project</a> enables safe
+          sharing of human genetic variants.<br>
+          <br>
+          Galaxy lets you use the Beacon protocol to share genetic variants directly from your analysis with the scientific
+          community in the following anonymous way:<br>
+          <br>
+          For participating users, we will merge variant lists to be shared into a single Beacon dataset and make that
+          dataset accessible through a Beacon server.<br>
+          If someone queries the server for a specific variant that is in our Beacon dataset, the server will reply with
+          <span class="cursive">“Yes, we have seen such a variant”</span>. Nothing more.<br>
```

Comment:
The "Nothing more." in this line is actually a lie.

We do share the _allelecount_ and_callcount_ as described in the novel i recently wrote (https://github.com/usegalaxy-eu/galaxy/pull/131#discussion_r1005540877)

Reply:
I removed the "Nothing more." which was not really adding much to the explanation anyways :sun_with_face: